image_classification/FF_Only/README.md

```diff
@@ -0,0 +1,23 @@
+# Pay Attention to MLPs, [arxiv](https://arxiv.org/abs/2105.02723) , [code](https://github.com/lukemelas/do-you-even-need-attention)
+
+## Usage
+
+```python
+from config import get_config
+from ffonly import build_ffonly
+# config files in ./configs/
+config = get_config('./configs/FF_Base.yaml')
+# build model
+model = build_ffonly(config)
+# load pretrained weights, .pdparams is NOT needed
+model_state_dict = paddle.load('/home/aistudio/data/data96150/linear_base.pdparams')
+model.set_dict(model_state_dict)
+```
+
+## Models Zoo
+
+|  Model  | Top-1 Acc. | Top-5 Acc. |                             Link                             |
+| :-----: | :--------: | :--------: | :----------------------------------------------------------: |
+| FF_Base |   0.746    |   0.916    | [AI Studio](https://aistudio.baidu.com/aistudio/datasetdetail/96150) |
```

Comment:
The result seems not consistent  with official results, and my test is also not the same:

Model    Official    MyTest    Current README
tiny         61.4 	     58.24   	60.9
base	74.9 	73.86 	74.6

Reply:
Now the updated results are: 
	Model Official MyTest  your README
	tiny 	61.4 	61.28 	60.9
	base	74.9 	74.82 	74.6

The above results now seems consistent with official results.

Note that the data norm has same mean and std to the MLP-Mixer.

Please update the readme: 

Tiny: Validation Acc@1: 0.6128, Validation Acc@5: 0.8406
Base: Validation Acc@1: 0.7482, Validation Acc@5: 0.9171